catalyst/contrib/nn/modules/arcface.py

```diff
@@ -76,7 +76,9 @@ def __repr__(self) -> str:
             + ")"
         )
 
-    def forward(self, input, target):
+    def forward(
+        self, input: torch.FloatTensor, target: torch.LongTensor
+    ) -> torch.FloatTensor:
```

Comment:
can we use `torch.Tensor` instead of `torch.FloatTensor` here?
If we use AMP (and HalfTensors) everything will be fine?